BiasAndFairnessModule/src/eval_engine/eval_runner.py

```
@@ -46,12 +49,24 @@ def evaluate_fairness(X, y, A, model):
+    # Calculate fairness metrics separately
+    fairness_metrics = {}
+    fairness_metrics["demographic_parity"] = demographic_parity_difference(
+        y_true=y_test, y_pred=y_pred, sensitive_features=A_test
+    )
+    fairness_metrics["equalized_odds"] = equalized_odds_difference(
+        y_true=y_test, y_pred=y_pred, sensitive_features=A_test
+    )
+    
     # Extract overall differences
     group_metrics = {}
     for metric_name in metrics.keys():
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)